server/manifest.go

```diff
@@ -6,5 +6,5 @@ import "github.com/mattermost/mattermost-server/v5/model"
 
 var manifest = model.Manifest{
 	Id:      "com.mattermost.msteamsmeetings",
-	Version: "2.0.0",
+	Version: "2.1.0",
```

Comment:
Why is `webapp/src/manifest.ts` not also updated?

Reply:
Because it doesn't have to:
https://github.com/mattermost/mattermost-plugin-msteams-meetings/blob/1b2818c8f5fec897cae113c42fcc63aaf6475707/webapp/src/manifest.ts#L1-L8